src/components/navigation-header/navigation-header.module.css

```diff
@@ -105,24 +93,24 @@
 
 	/* properties */
 	background-color: transparent;
-	color: var(--token-color-foreground-high-contrast);
+	color: var(--token-color-foreground-strong);
 
 	&:hover,
 	&[aria-expanded='true'] {
-		border-color: var(--token-color-palette-neutral-200);
+		border-color: var(--token-color-border-strong);
 	}
 
-	@nest html[data-theme='light'] & {
+	/* @nest html[data-theme='light'] & {
 		&:active {
 			background-color: var(--token-color-foreground-primary);
 		}
-	}
+	} */
 
-	@nest html[data-theme='dark'] & {
+	/* @nest html[data-theme='dark'] & {
 		color: var(--token-color-foreground-strong);
 
 		&:hover, &[aria-expanded='true'] {
 			border-color: var(--token-color-border-strong);
 		}
-	}
+	} */
```

Comment:
Why do we need to comment this code out?

Reply:
Oh, I was testing whether we needed that code, and left that in before review 😅 I'll double-check what's needed from it and remove in the next commit if I can (I should be able to) :+1:

Reply:
@nandereck Removed the comments as of this commit (they were safe to remove):
eaf25cf